app/src/main/java/com/infomaniak/drive/ui/fileList/fileShare/FileShareAddUserDialog.kt

```diff
@@ -105,7 +107,7 @@ class FileShareAddUserDialog : FullScreenBottomSheetDialog() {
             selectedPermission = bundle.getParcelable(PERMISSION_BUNDLE_KEY)!!
         }
 
-        shareButton.initProgress(this)
+        shareButton.initProgress(this@FileShareAddUserDialog)
         shareButton.setOnClickListener {
```

Comment:
Use `shareButton.apply { … }`

Reply:
Let's do this kind of improvement in another PR where regroup everything